ts-sdk/whirlpool/src/token.ts

```diff
@@ -33,6 +32,8 @@ import {
   getCreateAccountWithSeedInstruction,
   getTransferSolInstruction,
 } from "@solana-program/system";
+import { getTokenSize } from "@solana-program/token";
+import { getTokenSize as getToken22Size } from "@solana-program/token-2022";
```

Comment:
nit: maybe getTokenWithExtensionsSize since if it is a t22 token with no extensions we still the other one